sdcm/nemesis.py

```diff
@@ -4422,6 +4438,7 @@ def run_write_scylla_bench_load(write_cmd):
                     regex=".*sstable - Error while linking SSTable.*filesystem error: stat failed: No such file or directory.*"):
                 write_thread = self.tester.run_stress_thread(stress_cmd=write_cmd, stop_test_on_failure=False)
                 self.tester.verify_stress_thread(write_thread)
+                self.stop_nemesis_on_stress_errors(write_thread)
```

Comment:
Here we run the same command on each of the existing loaders.
In this case we are totally ok to get `n_loaders - 1` failures.
We need result at least from some of them.

Probably it is the case for some other nemesis.

It runs a single command on multiple loaders just because of the default parameter `round_robin=False` defined in the `self.tester.run_stress_thread`.

Reply:
Changed the check in the new method so that a nemesis would be aborted if all 'n_loaders` stress commands failed. Checked this for both cases - `round_robin` enabled and disabled.